### PR TITLE
Create gmail package

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull-request for issue #[15694](https://github.com/slingr-stack/platform/issues/15694) created by @agreggio-slingr 

## What it does?

Create the Gmail package

## How to test it?

You can try this on the following implementation on Idea2:
https://testpackages.idea2.io/dev/builder/login.html?sso=true&_k=builder_xKX7wDrVnb

Login
```javascript
pkg.gmail.api.getAccessToken();
```

Test

```javascript
var email='example@gmail.com';
log(JSON.stringify(pkg.gmail.api.get('/users/'+email+'/profile')));
```

## Implementation notes

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,23 +19,29 @@
...
-You can make `GET`,`PUT`,`PATCH`,`DELETE` requests to the [skeleton API](API_URL_HERE) like this:
+You can make `GET`,`PUT`,`PATCH`,`DELETE` requests to the [Gmail API](https://developers.google.com/gmail/api/guides) like this:
 ```javascript
-var response = pkg.skeleton.api.get('/path3')
-var response = pkg.skeleton.api.put('/path1/:testPath', body)
-var response = pkg.skeleton.api.put('/path1/:testPath')
-var response = pkg.skeleton.api.patch('/path2?param2=' + httpOptions.query.param2 + '&param3=' + httpOptions.query.param3 + '', body)
-var response = pkg.skeleton.api.patch('/path2?param2=' + httpOptions.query.param2 + '&param3=' + httpOptions.query.param3 + '')
-var response = pkg.skeleton.api.delete('/path4')
+var response = pkg.gmail.api.get('/path3')
```
Comment (pasaperez-slingr) [inline README.md:30]: Use the paths of the gmail api, you can use the same path of the test example of this PR.

File: flowSteps/apiCallGmail/step.js [hunk 2]
```diff
@@ -117,91 +113,48 @@ function stringToObject (obj) {
...
+	});
+	options.authorization = authorization;
+	return options;
 }
 
-function mergeJSON (json1, json2) {
-	var result = {};
-	var key;
+function mergeJSON(json1, json2) {
+	const result = {};
```
Comment (pasaperez-slingr) [inline flowSteps/apiCallGmail/step.js:151]: Remember avoid use ES6 features on flowSteps scripts

File: package.json [hunk 3]
```diff
@@ -11,65 +11,69 @@
         {
             "type": "package",
             "name": "oauth",
-            "version": "v1.0.19"
+            "version": "v1.0.24"
```
Comment (pasaperez-slingr) [inline package.json:14]:
```suggestion
            "version": "^v1.0.24"
```

**Review by pasaperez-slingr — APPROVED** [review]
Everything works as expected

File: package.json (hunk 3 above)
Comment (agreggio-slingr) [reply package.json:14]: I prefer not to add automatic updates

**Pull request merged**